packages/components/package.json

```diff
@@ -16,46 +16,36 @@
   },
   "license": "MPL-2.0",
   "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
   "scripts": {
-    "build": "ember build --environment=production",
+    "build": "rollup --config",
+    "prepublishOnly": "rollup --config",
+    "start": "rollup --config --watch",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "lint:css": "stylelint \"app/styles/**/*.scss\"",
+    "lint:css": "stylelint \"src/styles/**/*.scss\"",
     "lint:css:fix": "npm-run-all \"lint:css --fix\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "lint:types": "tsc --noEmit",
-    "start": "ember serve",
-    "test": "npm-run-all lint test:*",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
-    "test:ember:percy": "percy exec ember test",
-    "test:a11y": "ENABLE_A11Y_AUDIT=true ember test --server",
-    "test:a11y-report": "ENABLE_A11Y_MIDDLEWARE_REPORTER=true ember test"
+    "lint:types": "tsc --noEmit"
   },
   "dependencies": {
     "@ember/render-modifiers": "^2.0.5",
     "@ember/string": "^3.1.1",
     "@ember/test-waiters": "^3.1.0",
+    "@embroider/addon-shim": "^1.8.7",
     "@hashicorp/design-system-tokens": "^1.9.0",
     "@hashicorp/ember-flight-icons": "^4.0.4",
     "dialog-polyfill": "^0.5.6",
     "ember-a11y-refocus": "^3.0.2",
-    "ember-auto-import": "^2.6.3",
     "ember-cached-decorator-polyfill": "^1.0.2",
-    "ember-cli-babel": "^8.2.0",
-    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-sass": "^11.0.1",
```

Comment:
I am pretty sure this addon has no effect in v2 format as we take care of sass compilation with rollup pipeline

Reply:
Good call, yes it should be safe to remove it. Will amend